lib/githubbot.ts

```diff
@@ -251,7 +252,7 @@ export class GithubBot extends ProcBot.ProcBot<string> {
                         // If there's more retries, try again in 5 seconds.
                         setTimeout(runApi, 5000);
                     }
-                }).then(resolve);
+                });
             };
```

Comment:
Good catch! There was actually a lot wrong with the method in general, and I've altered it, which will be in next push (in a sec!). I've not moved to async yet, I've never used it in anger and I'd like to understand it properly before using it. I'm going to add this as an issue which I can correct once the panic of this week is over, if that's ok? :)